### PR TITLE
Fix some stupid things

### PR DESCRIPTION
**Pull request by auravoid**
:p

### CODE REVIEW

**Comment by auravoid** [comment]
HUH 

how did i add lines
image

**Comment by auravoid** [comment]
2c211d0 fixed the following:

> HUH
> 
> how did i add lines image

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # floatingmilkshake.com
+
```
Comment (auravoid) [inline README.md:2]: best part of this commit

**Comment by FloatingMilkshake** [comment]
lol thanks

**Pull request merged**